packages/experimental/ActivityIndicator/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@fluentui-react-native/experimental-activity-indicator",
+  "version": "0.1.0",
+  "description": "A cross-platform Fluent Activity Indicator component",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "verify-api": "fluentui-scripts verify-api-extractor",
+    "update-api": "fluentui-scripts update-api-extractor"
+  },
+  "dependencies": {
+    "@fluentui-react-native/framework": "0.5.2",
+    "@fluentui-react-native/tokens": ">=0.9.0 <1.0.0"
+  },
+  "devDependencies": {
+    "@types/react-native": "^0.63.0",
+    "@uifabricshared/build-native": "^0.1.1",
+    "@uifabricshared/eslint-config-rules": "^0.1.1",
+    "react-native": "^0.63.4"
+  },
+  "peerDependencies": {
+    "react-native": ">=0.63.4"
```

Comment:
Nit: One is "^", the other ">=". I think they both effectively resolve to the same thing (cheatsheet here https://devhints.io/semver) for versions less than 1.x.x.

But we should use the same symbols. It's especially concerning if you copied this from another package because it means we should go fix that other package and any other packages that differ.

Reply:
I just looked through the other packages and they all have some of each style so I think it's okay, but I agree that using a consistent notation would be nice.

Reply:
That's fair. For the react-native Dependency sepcifically, I think we try to be more strict. we can leave this as a followup since I think it doesn't have any real effect.